docs/en/observability/synthetics-configuration.asciidoc

```diff
@@ -82,7 +82,48 @@ For all available options, refer to the https://playwright.dev/docs/test-configu
 Do not attempt to run in headful mode (using `headless:false`) when running through Elastic's global managed testing infrastructure or Private Locations as this is not supported.
 ====
 
-Below are details on a few Playwright options that are particularly relevant to Elastic Synthetics including timeouts, timezones, and device emulation.
+Below are details on a few Playwright options that are particularly relevant to Elastic Synthetics including TLS client authentication, timeouts, timezones, and device emulation.
+
+[discrete]
+[[synthetics-configuration-playwright-options-client-certificates]]
+== TLS client authentication
+To enable TLS client authentication, set https://playwright.dev/docs/api/class-testoptions#test-options-client-certificates[`clientCertificates`] option in the configuration file:
+
+[source,js]
+----
+playwrightOptions: {
+  clientCertificates: [
+    {
+      origin: 'https://example.com',
+      certPath: './cert.pem',
```

Comment:
Can you call out the Cert path, key paths works only on Private locations and not on our managed Locations? and running with those would throw error during push workflow.